firefox-ios/Client/Frontend/Home/TopSites/DataManagement/UnifiedAds/UnifiedAdsConverter.swift

```diff
@@ -0,0 +1,23 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+/// To keep the integration of unified ads simple with minimal changes, we are converting Unified tiles
+/// into the old API Contile object. This will avoid logic changes inside the TopSitesDataAdaptor, which will be deprecated
+/// soon anyway due to the refactor of the homepage.
+struct UnifiedAdsConverter {
+    func convert(unifiedTiles: [UnifiedTile]) -> [Contile] {
```

Comment:
This could be static
```suggestion
    static func convert(unifiedTiles: [UnifiedTile]) -> [Contile] {
```